aws-analytics-pinpoint/src/main/java/com/amplifyframework/analytics/pinpoint/AutoSessionTrackerOld.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.analytics.pinpoint;
+
+import android.app.Activity;
+import android.app.Application;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.amazonaws.mobileconnectors.pinpoint.analytics.AnalyticsClient;
+import com.amazonaws.mobileconnectors.pinpoint.analytics.SessionClient;
+
+/**
+ * Tracks when the host application enters or leaves foreground.
+ * The constructor registers to receive activity lifecycle events.
+ **/
+final class AutoSessionTrackerOld implements Application.ActivityLifecycleCallbacks {
```

Comment:
This file will be deleted after mobileclient dependency is removed in the upcoming PR

Reply:
Can we name is AutoSessionTrackerLegacy